tests/e2e/scale/noobaa/test_perf_mcg.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import E2ETest, scale
+from ocs_ci.framework.pytest_customization.marks import vsphere_platform_required
+from ocs_ci.ocs import hsbench
+from ocs_ci.ocs.mcg_workload import get_pod_logs
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def s3bench(request):
+    # Create hs s3 benchmark
+    s3bench = hsbench.HsBench()
+    s3bench.create_resource_hsbench()
+    s3bench.install_hsbench()
+
+    def teardown():
+        s3bench.cleanup()
+
+    request.addfinalizer(teardown)
+    return s3bench
+
+
+@scale
+class TestNoobaaPerformance(E2ETest):
+    """
+    Noobaa Performance in the Multicloud Object Gateway (MCG) and
+    Ceph Object Gateway (RGW)
+
+    """
+
+    @pytest.mark.polarion_id("OCS-2577")
+    def test_perf_mcg_fio(self, mcg_obj, bucket_factory, mcg_job_factory):
+        """
+        Create 2 jobs using mcg_job_factory to run at the same time.
+        These jobs are created and used FIO workload through mcg_job_factory.
+        This test case is created using the same scenario of:
+        https://bugzilla.redhat.com/show_bug.cgi?id=1968322#
+
+        """
+        run_time = 3600
```

Comment:
This test case takes more than 4hrs to complete with the set of arguments that defined in the test case.
Using s3cmd sync to run fio, it only takes ~10 mins to complete.  I need to revisit this test case if it can be run as performance test case.